src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -470,4 +470,70 @@ private function writeText(string $content, array $options = array())
             isset($options['raw_output']) ? !$options['raw_output'] : true
         );
     }
+
+    private function describeRouteMethods(array $methods)
+    {
+        if (!$methods) {
+            return '<info>ANY</info>';
+        }
+
+        $knownMethods = array(
```

Comment:
We could use `Request::METHOD_GET`, `Request::METHOD_POST`, etc. By the way, Symfony's Request also supports `PURGE` method.